amundsen_application/static/js/components/ColumnList/index.spec.tsx

```diff
@@ -96,6 +113,27 @@ describe('ColumnList', () => {
 
         expect(actual).toEqual(expected);
       });
+
+      describe('when sorting is passed', () => {
+        it('should sort the data by that value', () => {
+          const { wrapper } = setup({
+            columns,
+            sortBy: {
+              key: 'usage',
+              direction: SortDirection.descending,
+              name: 'Usage',
+            },
+          });
+          const expected = 'simple_column_name_timestamp';
```

Comment:
So here are you testing if it was sorted correctly by checking just the first item to see if its the one that is supposed to be first?